legacy/src/app/routes.js

```diff
@@ -19,6 +19,9 @@ import vlanDetailsTmpl from "./partials/vlan-details.html";
 import zoneDetailsTmpl from "./partials/zone-details.html";
 import zonesListTmpl from "./partials/zones-list.html";
 
+const prefixRoute = (route) =>
```

Comment:
Unfortunately, afaict, angularjs provides no way to programatically specifiy a basepath when using the locationProvider's html5mode (it expects to be defined in `index.html` which is not possible in the singlespa context, which is unfortunately why this code exists.